contentful/apps
# feat: trigger save from app config notification section [INTEG-1715]

## Purpose

This PR implements the new save methods defined in experience packages / user interface to trigger a save of the app from the app configuration location.

## Approach

The new `save` and `quickSave` methods are added to the `SdkWithCustomApiProvider` and then called when a user adds, edits, or deletes a notification. This PR also updates the copy of the `save` button to be either `create` or `update` depending on whether the notification is new. Also, I removed all the old notifications reminding a user to save, and am using the new `save` method to persist the tenant id after a user authenticates with Microsoft.

## Testing steps

Run the MS Teams frontend locally. Install the app and authenticate with Microsoft, seeing the app installs automatically after you authenticate. Then create, edit, and delete notifications to see the quick save.

## Breaking Changes

## Dependencies and/or References

## Deployment
